aws/resource_aws_cloudwatch_dashboard_test.go

```diff
@@ -75,6 +75,36 @@ func TestAccAWSCloudWatchDashboard_update(t *testing.T) {
 	})
 }
 
+func TestAccAWSCloudWatchDashboard_updateName(t *testing.T) {
+	var dashboard cloudwatch.GetDashboardOutput
+	rInt := acctest.RandInt()
+	rInt2 := acctest.RandInt()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCloudWatchDashboardDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCloudWatchDashboardConfig(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudWatchDashboardExists("aws_cloudwatch_dashboard.foobar", &dashboard),
```

Comment:
Likely copy-paste from the earlier iteration of this pull request, the test configuration resource name is `aws_cloudwatch_dashboard.test`:

https://github.com/terraform-providers/terraform-provider-aws/blob/e1fc857056e66b257520f73ca5e22dc6facaebef/aws/resource_aws_cloudwatch_dashboard_test.go#L159

Otherwise, this test causes this error:

```
--- FAIL: TestAccAWSCloudWatchDashboard_updateName (5.64s)
    testing.go:640: Step 0 error: Check failed: Check 1/3 error: Not found: aws_cloudwatch_dashboard.foobar
```

Since it is a small testing fix, will do so on merge. 👍

Reply:
Gah, I was so close. Thanks!